pkg/styra/workspace.go

```diff
@@ -68,7 +69,15 @@ func (c *Client) UpdateWorkspace(
 	ctx context.Context,
 	request *UpdateWorkspaceRequest,
 ) (*UpdateWorkspaceResponse, error) {
-	res, err := c.request(ctx, http.MethodPut, endpointV1Workspace, request)
+	return c.UpdateWorkspaceRaw(ctx, request)
+}
+
+// UpdateWorkspaceRaw calls the PUT /v1/workspace endpoint in the Styra API.
```

Comment:
```suggestion
// UpdateWorkspaceRaw calls the PATCH /v1/workspace endpoint in the Styra API.
```

Same change to line 67 (description for UpdateWorkspace